front/src/modules/trainschedule/components/TimetableV2/TimetableToolbar.tsx

```diff
@@ -181,13 +180,23 @@ const TimetableToolbar = ({
           )}
         </div>
         <div>
-          {multiSelectOn && <span>{selectedTrainIds.length} / </span>}
-          {t('trainCount', {
-            count: trainSchedulesDetails.length,
-          })}
+          {trainSchedules.length > 0
```

Comment:
it’s a bit confusing to have the same box used to display the number of trains that are selected and the number of train loaded.
we Could display the <Loader /> component (in really small) while it’s loading to make it clear. and even write (loading 10/156 trains … )

Reply:
do we display a <Loader /> on top of the GET aswell ?

Reply:
Yes some UI improvements are planned for a next PR, can I fix your comment in it ?

Reply:
sure